app/views/auth/login.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Log In</h1>
+<% if @errors %>
```

Comment:
we created an errors partial for all new/edit forms that will be rendered HOWEVER the login error is only a string and needs its own logic: 

made partials [development dfa4ee7]